libraries/AC_WPNav/AC_WPNav.cpp

```diff
@@ -619,14 +680,24 @@ bool AC_WPNav::force_stop_at_next_wp()
 
 // get terrain's altitude (in cm above the ekf origin) at the current position (+ve means terrain below vehicle is above ekf origin's altitude)
 bool AC_WPNav::get_terrain_offset_cm(float& offset_cm)
+{
+    // call meter-version and convert to centimeters
+    float offset_m = offset_cm;
+    const bool ret = get_terrain_offset_m(offset_m);
+    offset_cm = offset_m * 100.0;
+    return ret;
+}
+
+// get terrain's altitude (in cm above the ekf origin) at the current position (+ve means terrain below vehicle is above ekf origin's altitude)
```

Comment:
we should update the comment to say "(in meters above the ekf origin)"

Reply:
// Returns terrain offset in cm above the EKF origin at the current position.
// See get_terrain_offset_m() for full details.